### PR TITLE
Add dummy backend test to run checker on all model and node tests

### PR DESCRIPTION
**Pull request by dzhulgakov**
~~The tests themselves actually fail right now for 2 reasons:~~
~~1) renamed SpatialBN -> BatchNorm~~
~~2) LRN loosing it's second parameter~~

### CODE REVIEW

File: onnx/backend/test/runner/__init__.py [hunk 1]
```diff
@@ -145,13 +145,22 @@ def _prepare_model_data(self, model_test):
...
+            download_file = tempfile.NamedTemporaryFile(delete=False)
+            try:
+                download_file.close()
                 print('Start downloading model {} from {}'.format(
                     model_test.model_name, url))
                 urlretrieve(url, download_file.name)
                 print('Done')
                 with tarfile.open(download_file.name) as t:
                     t.extractall(models_dir)
+            finally:
```
Comment (anderspapitto) [inline onnx/backend/test/runner/__init__.py]: shouldn't this be an `except` block rather than a `finally` block? @dzhulgakov @bddppq

Comment (bddppq) [reply onnx/backend/test/runner/__init__.py]: `NamedTemporaryFile(delete=True)` (the original behavior) was going to delete the temp file no matter the download succeeded or not.

Comment (anderspapitto) [reply onnx/backend/test/runner/__init__.py]: sure, but it wasn't going to lie about the test failing

Comment (bddppq) [reply onnx/backend/test/runner/__init__.py]: Oh the print line, I was thinking about the `os.remove` line. Yes you are right, that should belong to `except` block

**Pull request merged**